is_core/forms/models.py

```diff
@@ -221,20 +222,22 @@ class SmartModelForm(six.with_metaclass(SmartModelFormMetaclass, SmartFormMixin,
 
     def __init__(self, *args, **kwargs):
         # Set values must be ommited
-        readonly_exclude = kwargs.get('initials', {}).keys()
+        readonly_exclude = kwargs.get('initials',
+                                      {}).keys()
 
         super(SmartModelForm, self).__init__(*args, **kwargs)
 
         opts = self._meta
         self.humanized_data = humanized_model_to_dict(self.instance, self.base_readonly_fields, opts.fields,
-                                                      itertools.chain(opts.exclude or (), readonly_exclude or  ()))
+                                                      itertools.chain(opts.exclude or (), readonly_exclude or ()))
 
         if self.instance.pk:
             for rel_object in opts.model._meta.related_objects:
                 if rel_object.name in self.fields and rel_object.name not in self.initial:
-                    self.initial[rel_object.name] = list(
-                        getattr(self.instance, rel_object.name).values_list('pk', flat=True)
-                    )
+                    if hasattr(self.instance, rel_object.name):
```

Comment:
I looked at Demand form today and 'communication' is a form field of the DemandForm, not a model field. But I have to specify it in form_fields on the DemandIsCore because there are some properties (buttons) on the model and I cannot specify those on DemandForm. Or can I? @matllubos